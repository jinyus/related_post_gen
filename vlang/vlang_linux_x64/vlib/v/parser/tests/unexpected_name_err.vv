fn main() {
	asfasfasfsfa
	println('text')
}
