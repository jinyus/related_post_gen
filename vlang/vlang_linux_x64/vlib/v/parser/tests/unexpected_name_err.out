vlib/v/parser/tests/unexpected_name_err.vv:2:2: error: unexpected name `asfasfasfsfa`
    1 | fn main() {
    2 |     asfasfasfsfa
      |     ~~~~~~~~~~~~
    3 |     println('text')
    4 | }
